.travis.yml

```diff
@@ -1,17 +1,9 @@
 language: erlang
 otp_release:
-  - 17.1
   - 17.0
   - R16B03-1
-  - R16B03
```

Comment:
Should we not keep these, as well? I am guessing not everyone always runs on the latest release.